airflow/models/abstractoperator.py

```diff
@@ -101,6 +101,7 @@ class AbstractOperator(LoggingMixin, DAGNode):
 
     owner: str
     task_id: str
+    is_mapped: ClassVar[bool]
```

Comment:
@uranusjr -> maybe there is another way, but I believe the lack of "is_mapped"  in AbstractOperator was a bit wrong (and it would prevent adding the proper typing in task_group_to_grid ad AbstractOperator missed `is_mapped` field.